index.js

```diff
@@ -84,14 +84,14 @@ var layerTypes = [
     name: {
       en: 'board outline'
     },
-    match: /(Edge.Cuts)|(\.((dim$)|(mil$)|(gm[l\d]$)|(gko$)|(fab$)))|(\.boardoutline\.ger$)|(\.outline\.gbr$)/i
+    match: /(Edge.Cuts)|(\.((dim$)|(mil$)|(gm[l\d]$)|(gko$)|(fab$)))|(\.boardoutline\.ger$)|(\.outline\.gbr$)|(mechanical \d+\.txt$)/i
   },
   {
     id: 'drl',
     name: {
       en: 'drill hits'
     },
-    match: /\.((fab\.gbr$)|(cnc$)|(drl$)|(xln$)|(txt$)|(tap$)|(drd$)|(exc$))/i
+    match: /(drill\.txt$)|(\.((fab\.gbr$)|(cnc$)|(drl$)|(xln$)|(txt$)|(tap$)|(drd$)|(exc$)))/i
```

Comment:
Is this necessary given that `.txt$` is already a matcher?

Reply:
No, I'll remove it. I was thinking about how to better handle the .txt though as it looks like Proteus also generates a README.txt by default. I can't think of anything that doesn't require actually trying to parse the file though.